### ISSUE TITLE
Latest lambci/docker-lambda image has removed hardcoded default credentials

### ISSUE THREAD

**Issue by j-sv** [localstack/localstack#2819]
# Type of request: This is a ...

[*] bug report
[ ] feature request

# Detailed description

Perhaps not a bug report per se, but this affected me at least. A new version of `lambci/docker-lambda` was pushed recently. [This commit](https://github.com/lambci/docker-lambda/commit/7ece2742a5b0e84eb09d6ed659f123474e619b27) removed hardcoded defaults for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.

I have a lambda that reads a value from secret in `secretsmanager`. If I don't set dummy values for `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` when creating the lambda invoking it will time out reading the secret.

## Expected behavior

A lambda function can read a secret from `secretsmanager`.

## Actual behavior

Lambda times out trying to read secret.

# Steps to reproduce

Create a lambda which tries to read a secret from `secretsmanager` without adding the environments variables mentioned above.

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

This will work:
```bash
awslocal lambda create-function \
  --function-name foo \
  --runtime go1.x \
  --handler main \
  --role arn:aws:iam::123456790:role/ignore \
  --zip-file fileb://foo.zip \
  --environment "Variables={FOO=bar,AWS_ACCOUNT_ID=123456790,AWS_ACCESS_KEY_ID=0,AWS_SECRET_ACCESS_KEY=0}"
```

This will time out while reading:
```bash
awslocal lambda create-function \
  --function-name foo \
  --runtime go1.x \
  --handler main \
  --role arn:aws:iam::123456790:role/ignore \
  --zip-file fileb://foo.zip \
  --environment "Variables={FOO=bar}"
```

**Comment by whummer** [comment]
Thanks for reporting @j-sv . This should be fixed in #2829 - can you please pull the Docker image again and give it another try with the `latest` version? Please report here if the problem persists. Thanks

**Comment by j-sv** [comment]
Thanks @whummer, @macnev2013 ! I've verified that this works with the `latest` Docker image.
